theme_bookstore/views/snippets/s_masonry_block.xml

```diff
@@ -4,70 +4,69 @@
 <template id="s_masonry_block_default_template" inherit_id="website.s_masonry_block_default_template">
```

Comment:
You forgot the customization in the `theme_vehicle/.../customization.xml` file :wink: 
You should set the container as `container-fluid` and it seems to not target the right elements when changing the texts :+1: 
![image](https://github.com/user-attachments/assets/363a6949-4787-4f57-89ce-ec6c600c9ea2)